docs/api-references/order.md

```diff
@@ -0,0 +1,53 @@
+## Order - API
+
+### `POST` Add New Order
+```
+api/orders/create
```

Comment:
Please check this document: https://laravel.com/docs/5.4/controllers#resource-controllers
URI should be `api/orders/`